src/Model/README.md

```diff
@@ -0,0 +1,5 @@
+## Friendica\Model
+
+Models are the glue between the business logic of the app and the datastore(s).
+
+In the namespace Model should only be static classes that interact with the DB with the same name as a database table.
```

Comment:
I was planning on making Models actual objects roughly holding a table row, what do you think about it?

Reply:
This sounds partly like [ORM](https://en.wikipedia.org/wiki/Object-relational_mapping#Object-oriented_databases). Which would be fine for me, because in the end, I think we should reduce the direct `DBA` calls at nearly every place in the code.

But I'd like to discuss it in the dev-forum :-)

Reply:
A quote:
> The programmer must either convert the object values into groups of simpler values for storage in the database (and convert them back upon retrieval), or only use simple scalar values within the program. 

Currently, we're mixing both. The `Model` is part of the first paradigm, the different arrays, we're using based on DB-calls is part of the second paradigm.

Reply:
Ok, I just wanted to know your opinion about ORM. The tricky part with ORM is the `n ↔ m` relationship and recursive foreign keys, so I wanted to be sure I wasn't the only one wanting to go this route.